lib/snapshot/reports_generator.rb

```diff
@@ -13,16 +13,20 @@ def generate
       Dir[File.join(screens_path, "*")].sort.each do |language_folder|
         language = File.basename(language_folder)
         Dir[File.join(language_folder, '*.png')].sort.each do |screenshot|
-          available_devices.each do |key_name, output_name|
-            next unless File.basename(screenshot).include?(key_name)
+       	   ["portrait", "landscape"].each do |orientation|	
```

Comment:
Use 2 (not 3) spaces for indentation.<br>Tab detected.<br>Trailing whitespace detected.